### PR TITLE
feat: (sfui2-1239) prose styling

### PR DESCRIPTION
**Pull request by aniamusial**
# Related issue

https://vsf.atlassian.net/browse/SFUI2-1239?atlOrigin=eyJpIjoiOWU5ZjVlNmE4OWYxNDJhYzlhOWYxMDY3Mzg3MGM3MWEiLCJwIjoiaiJ9

# Scope of work

- installed `tailwind/typography` plugin
- updated docs with examples of `prose` styling
- updated import of fonts with italic styles

# Screenshots of visual changes

Zrzut ekranu 2023-07-4 o 10 53 50
Zrzut ekranu 2023-07-4 o 10 53 57
Zrzut ekranu 2023-07-4 o 10 54 04

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x] cypress tests created

### CODE REVIEW

File: apps/preview/next/package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "@storefront-ui/tests-shared": "workspace:*",
     "@storefront-ui/tw-plugin-peer-next": "workspace:*",
     "@storefront-ui/typography": "workspace:*",
+    "@tailwindcss/typography": "^0.5.9",
```
Comment (Szymon-dziewonski) [inline apps/preview/next/package.json:35]: question: do we need to install this plugin if we have our own typography plugin ? I'm not sure how it is working

File: packages/config/tailwind/index.ts [hunk 2]
```diff
@@ -9,6 +11,207 @@ export const tailwindConfig: Config = {
   },
   theme: {
     extend: {
+      typography: ({ theme }: ThemeConfig) => ({
```
Comment (Szymon-dziewonski) [inline packages/config/tailwind/index.ts:14]: question: I thought that OUR prose styles should be inside our plugin, so users would be able just to install our plugin and have our prose styling, I'm confused here a bit :)

**Review by Szymon-dziewonski — COMMENTED** [review]
Whole PR looks cool, good job, I left only 2 questions that are unclear for me, thanks

File: packages/config/tailwind/index.ts (hunk 2 above)
Comment (aniamusial) [reply packages/config/tailwind/index.ts:14]: Not sure if I understand correctly, do you mean - as part of SfTypography created by @FRSgit ?

Comment (FRSgit) [reply packages/config/tailwind/index.ts:14]: I think that one was made clear during the standup - is there anything else we can do to push this PR forward?

Comment (Szymon-dziewonski) [reply packages/config/tailwind/index.ts:14]: @FRSgit everything is clear and thank you and @aniamusial for explanation :)

**Review by Szymon-dziewonski — APPROVED** [review]

**Pull request merged**